src/main/java/duke/task/TaskManager.java

```diff
@@ -0,0 +1,174 @@
+package duke.task;
+
+import duke.exception.TodoFormatException;
+import duke.exception.DeadlineFormatException;
+import duke.exception.EventFormatException;
+import duke.exception.EmptyTasklistException;
+import duke.exception.DoneFormatException;
+import duke.exception.InvalidTaskIdException;
+import duke.exception.TaskAlreadyDoneException;
+import duke.exception.DeleteFormatException;
+
+import duke.ui.DukeInterface;
+
+import java.util.ArrayList;
+
+public class TaskManager {
+
+    public ArrayList<Task> tasks;
```

Comment:
To access this list outside the `TaskManager` class, it is better to define getter and setter methods (for example: `getTasks` that returns the list an `setTasks` that add the Tasks to the list.)